src/smc-util-node/misc_node.coffee

```diff
@@ -847,10 +847,10 @@ exports.loadSync = (filename) ->
     JSON.parse(fs.readFileSync(filename).toString())
 
 
-# WA state sales tax rates, as of August 2015.
-# Generated scripts/sales_tax.py
+# WA state sales tax rates, as of April 20 2016.
+# Generated via http://dor.wa.gov/Content/FindTaxesAndRates/RetailSalesTax/DestinationBased/ClientInterface.aspx
```

Comment:
1. Did you write a script to run through all zip codes and call the API a the URL, but **NOT** include that script with this patch?

2. Why are these rates different than the ones you included into Sage a few weeks ago?  Did you make typos, or what happened?